addons/danger/functions/fnc_shareInformation.sqf

```diff
@@ -35,8 +35,7 @@ _unit setVariable [QGVAR(currentTask), "Share Information"];
 
 // find units
 private _groups = allGroups select {
-    local _x
-    && {side _x isEqualTo side _unit}
+    && {(side _x) getFriend (side _unit) > 0.55}
```

Comment:
I would use https://community.bistudio.com/wiki/BIS_fnc_sideIsFriendly for this.

Reply:
aha! Thats the one. I'll fix it